{
  "artifacts": [],
  "command": "main",
  "experiment": {
    "base_dir": "/home/lgrootde/Generative-re-tests",
    "dependencies": [
      "datasets==2.17.1",
      "evaluate==0.4.1",
      "numpy==1.26.4",
      "sacred==0.8.5",
      "torch==2.2.1",
      "transformers==4.38.1",
      "wasabi==1.1.2"
    ],
    "mainfile": "run.py",
    "name": "generative_re",
    "repositories": [
      {
        "commit": "e3cd392d5ff25b8446103317d826f8b42ac902f2",
        "dirty": false,
        "url": "https://github.com/ontox-hu/Generative-re-tests"
      }
    ],
    "sources": [
      [
        "run.py",
        "_sources/run_dcefa1b70f5a4b0f3d111f9541f14fcc.py"
      ]
    ]
  },
  "fail_trace": [
    "Traceback (most recent call last):\n",
    "  File \"/home/lgrootde/Generative-re-tests/venv/lib/python3.12/site-packages/sacred/config/captured_function.py\", line 42, in captured_function\n    result = wrapped(*args, **kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^\n",
    "  File \"/home/lgrootde/Generative-re-tests/run.py\", line 587, in main\n    trainer = Seq2SeqTrainer(\n              ^^^^^^^^^^^^^^^\n",
    "  File \"/home/lgrootde/Generative-re-tests/venv/lib/python3.12/site-packages/transformers/trainer_seq2seq.py\", line 56, in __init__\n    super().__init__(\n",
    "  File \"/home/lgrootde/Generative-re-tests/venv/lib/python3.12/site-packages/transformers/trainer.py\", line 367, in __init__\n    self.create_accelerator_and_postprocess()\n",
    "  File \"/home/lgrootde/Generative-re-tests/venv/lib/python3.12/site-packages/transformers/trainer.py\", line 4166, in create_accelerator_and_postprocess\n    raise ValueError(f\"{wrapper} can't be used with `save_only_model` along with `load_best_model_at_end`.\")\n",
    "ValueError: DeepSpeed can't be used with `save_only_model` along with `load_best_model_at_end`.\n"
  ],
  "heartbeat": "2024-04-21T17:57:38.828609",
  "host": {
    "ENV": {},
    "cpu": "AMD EPYC 7V13 64-Core Processor",
    "gpus": {
      "driver_version": "545.23.08",
      "gpus": [
        {
          "model": "NVIDIA A100 80GB PCIe",
          "persistence_mode": false,
          "total_memory": 81920
        }
      ]
    },
    "hostname": "jupyternoteboo.src-azurefreebu.src.surf-hosted.nl",
    "os": [
      "Linux",
      "Linux-5.15.0-1059-azure-x86_64-with-glibc2.31"
    ],
    "python_version": "3.12.1"
  },
  "meta": {
    "command": "main",
    "config_updates": {},
    "named_configs": [
      "config/config_T5-11b_cdr.yaml"
    ],
    "options": {
      "--beat-interval": null,
      "--capture": null,
      "--comment": null,
      "--debug": false,
      "--enforce_clean": false,
      "--file_storage": null,
      "--force": false,
      "--help": false,
      "--id": null,
      "--loglevel": null,
      "--mongo_db": null,
      "--name": null,
      "--pdb": false,
      "--print-config": false,
      "--priority": null,
      "--queue": false,
      "--s3": null,
      "--sql": null,
      "--tiny_db": null,
      "--unobserved": false,
      "COMMAND": null,
      "UPDATE": [
        "config/config_T5-11b_cdr.yaml"
      ],
      "help": false,
      "with": true
    }
  },
  "resources": [],
  "result": null,
  "start_time": "2024-04-21T17:50:50.167485",
  "status": "FAILED",
  "stop_time": "2024-04-21T17:57:38.875478"
}